src/modules/avatar/avatar.controller.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  Controller,
+  Get,
+  Param,
+  Post,
+  Res,
+  UploadedFile,
+  UseInterceptors,
+} from '@nestjs/common';
+import { FileInterceptor } from '@nestjs/platform-express';
+import { join } from 'path';
+import { of } from 'rxjs';
+
+import { multerOptions } from '../../config/multer.config';
+
+@Controller('avatar')
+export class AvatarController {
+  @Post('upload')
+  @UseInterceptors(FileInterceptor('avatar', multerOptions))
+  photoUpload(@UploadedFile() file: Express.Multer.File) {
+    return `localhost:4000/avatar/${file.filename}`;
```

Comment:
please all links move to env or constants

Reply:
Moved to consts